### ISSUE TITLE
quick-reply: white background on browse button goes to mars in old browsers

### ISSUE THREAD

**Issue by nfagerlund** [dreamwidth/dw-free#2628]
Reported with Safari 9.1.2 (?!?!?!?!?!?!!)

https://dw-maintenance.dreamwidth.org/81236.html?thread=3215444#cmt3215444

Not sure what to do about that yet. A scorched-earth fix is best, but the trick is to make sure that fix doesn't affect any browsers that don't need it.

**Comment by kareila** [comment]
FWIW I get the correct positioning in Safari 10.1.2, so the affected rendering must have changed between 9 and 10.

**Comment by rahaeli** [comment]
Whereas I get the starburst and wrong positioning in the oldest version I have around (10.0.1), so it must be from a change in the minor version, and also I can reliably reproduce so hit me up for testing.

**Comment by kareila** [comment]
You're right, there was a major rendering engine update in 10.1. According to https://en.wikipedia.org/wiki/Safari_version_history#Safari_10 it was the first version to support CSS Grid Layout.

**Comment by rahaeli** [comment]
I looked a bit for it, but couldn't find it -- where is the button::after rule set, do you know?

**Comment by nfagerlund** [comment]
So you're in the web inspector already, yeah? (right-click -> inspect element)

Hmm, in the more recent browsers, the web inspector makes it easy to inspect ::after pseudo-elements directly (they appear as a child of the element they're attached to, so it would be inside the `<button>`), but safari 10 is probably too old for that. If you can't find it in the "elements" or "inspect" tab of the web inspector, you'll need to go into the main CSS. It looks like CURRENT safari calls that tab "Sources" -- there's a folder in its sidebar labeled "style sheets" that appears as a child of the current page, and then ONE of the CSS files in that folder should include a rule like this: 

`#qrformdiv .qr-icon a::after, #qrformdiv .qr-icon button::after {`

Since the CSS on prod is all compressed, you'll probably need to cmd-F for `::after`. Anyway, changing rules in there should still make changes live on the page you're inspecting.
